setuptools/config/_validate_pyproject/error_reporting.py

```diff
@@ -24,7 +24,7 @@
     "must not be there",
 )
 
-_NEED_DETAILS = {"anyOf", "oneOf", "anyOf", "contains", "propertyNames", "not", "items"}
+_NEED_DETAILS = {"anyOf", "oneOf", "contains", "propertyNames", "not", "items"}
```

Comment:
This code path is generated automatically, so we have to fix it first elsewhere, otherwise the changes can be overwritten in the next update...

```suggestion
_NEED_DETAILS = {"anyOf", "oneOf", "allOf", "contains", "propertyNames", "not", "items"}
```

Reply:
So do I commit the above suggestion or not? If the code is generated automatically, is it worth it?

Reply:
I think you can commit, I checked `validate-pyproject` and I think it was already corrected there. So if later we overwrite, that would not be problematic.

I am holding back a bit to re-generate the code because some minor changes were introduced. They should not cause problems, but you know how it is... Hyrum's Law hits heavily on the setuptools code base 😅.